atomic_reactor/utils/operator.py

```diff
@@ -734,6 +734,28 @@ def _replace_pullspecs_not_in_annotations(self, obj, k_or_i, replacements):
             # in replacement_pullspecs and no replacement will be done
             self._replace_unnamed_pullspec(obj, k_or_i, replacements)
 
+    def _mods_append(self, append_mods):
```

Comment:
I guess `mods` means modifications, right? Can we please use the full spelling to make it clear?

Reply:
Sure can